src/test/java/io/supertokens/test/passwordless/api/PasswordlessCreateCodeAPITest2_11.java

```diff
@@ -437,9 +437,7 @@ public void testMalformedDeviceId() throws Exception {
 
         assertNotNull(error);
         assertEquals(400, error.statusCode);
-        assertEquals(
-                "Http error. Status Code: 400. Message: Please provide exactly one of email, phoneNumber or deviceId",
-                error.getMessage());
+        assertEquals("Http error. Status Code: 400. Message: Input encoding error in DeviceId", error.getMessage());
```

Comment:
Error message should not have capital "D" in "DeviceId". Since the input is "deviceId". If this requires a change in the API, please do so in this PR.

Reply:
noted.